controllers/errors.controller.js

```diff
@@ -0,0 +1,16 @@
+  exports.handleCustomErrors = (err, req, res, next) => {
```

Comment:
These are some fine looking error handlers, but adding at this stage you're adding them in but not testing their behaviour!   Would've been better to add when you are going to use them...